assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItems.js

```diff
@@ -25,20 +25,57 @@ import PropTypes from 'prop-types';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { useState, useEffect } from '@wordpress/element';
 
 /**
- * External dependencies
+ * Internal dependencies
  */
-import { useSelect } from 'googlesitekit-data';
+import { useSelect, useDispatch } from 'googlesitekit-data';
 import { CORE_USER } from '../../../../../../googlesitekit/datastore/user/constants';
 import {
 	DATE_RANGE_OFFSET,
 	MODULES_ANALYTICS_4,
 } from '../../../../datastore/constants';
 import AudienceItem from './AudienceItem';
 import { SelectionPanelItems } from '../../../../../../components/SelectionPanel';
+import { CORE_UI } from '../../../../../../googlesitekit/datastore/ui/constants';
+import { AUDIENCE_SELECTION_PANEL_OPENED_KEY } from './constants';
+import AudienceItemPreviewBlock from './AudienceItemPreviewBlock';
 
 export default function AudienceItems( { savedItemSlugs = [] } ) {
+	const [ firstView, setFirstView ] = useState( true );
+	const { syncAvailableAudiences } = useDispatch( MODULES_ANALYTICS_4 );
+
+	const isOpen = useSelect( ( select ) =>
+		select( CORE_UI ).getValue( AUDIENCE_SELECTION_PANEL_OPENED_KEY )
+	);
+
+	const isLoading = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).isFetchingSyncAvailableAudiences()
+	);
+
+	useEffect( () => {
+		if ( ! firstView || ! isOpen ) {
+			return;
+		}
+
+		syncAvailableAudiences();
+		setFirstView( false );
+	}, [ firstView, isOpen, syncAvailableAudiences ] );
+
+	useEffect( () => {
+		// @TODO Explore more elegant option to re-establish the focus. After `syncAvailableAudiences`
+		// happens the focus is lost, even without preview block being shown.
```

Comment:
I opened a follow up issue #8968, to explore this a bit further, since it will go off the scope in current issue and take more time in this issue